package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "module",
+  "dependencies": {
+    "@datocms/cma-client-node": "^3.1.6",
+    "chalk": "^5.3.0",
+    "datocms-html-to-structured-text": "^3.0.0",
+    "dotenv": "^16.4.5",
+    "glob": "^10.3.12",
+    "jsonschema": "^1.4.1",
+    "mdast-util-from-markdown": "^2.0.0",
+    "mdast-util-to-hast": "^13.1.0",
+    "to-vfile": "^8.0.0",
+    "tsx": "^4.7.2",
```

Comment:
Ooh is this better than `ts-node`? It does make me think of jsx/tsx though.

Reply:
I liked it purely for having sensible defaults and not requiring me to add a `tsconfig.json` file.